sdk/src/time_stamp.rs

```diff
@@ -627,8 +583,19 @@ pub(crate) fn verify_timestamp(ts: &[u8], data: &[u8]) -> Result<TstInfo> {
             let validated_res: Result<bool> = if _sync {
                 #[cfg(feature = "openssl")]
                 {
-                    let validator = get_local_validator(sig_alg, hash_alg)?;
-                    validator.validate(&sig_val.to_bytes(), &tbs, &signing_key_der)
+                    let Some(validator) =
```

Comment:
IMHO this can all boil down to the synchronous version since we now have equivalent code for WASM. @cdmurph32 @mauricefisher64 any objection to removing the async path?

Reply:
Until we can support Es512 and others, then we need to keep the async path open. I'll leave the code as is.